### ISSUE TITLE
GCMPlayground doesn't load

### ISSUE THREAD

**Issue by azaman2204** [googlesamples/gcm-playground#18]
I can't seem to get GCMPlayground to load on my chrome browser using the my server IP as per docker server IP i.e in my case http://192.168.100.99:3000 although I can see the server output on the terminal. I can register from Android and Chrome.

**Comment by sanjaylod493** [comment]
I Am Poet Sir Please Share My Poetry Sir .
https://youtu.be/xD5ktAcGFrk

**Comment by sanjaylod493** [comment]
https://youtu.be/nWBgCt3PHeI

**Comment by silvolu** [comment]
@azaman2204 do you see any lines like:

```
web_1    | [BS] Access URLs:
web_1    |  ------------------------------------
web_1    |        Local: http://localhost:3000
web_1    |     External: http://172.17.0.24:3000
web_1    |  ------------------------------------
web_1    |           UI: http://localhost:3001
web_1    |  UI External: http://172.17.0.24:3001
web_1    |  ------------------------------------
web_1    | [BS] Serving files from: .tmp
web_1    | [BS] Serving files from: app
```

In the terminal where you started the playground?

**Comment by azaman2204** [comment]
Hi Silvano
Yes I saw that and try access using listed urls. I also tried replacing the
localhost with my local ip but not successful too.

In the terminal, I started playgroud from Applications/docker terminal


Kind regards;
Hj Azaman Hj Suleiman

**Comment by azaman2204** [comment]
Hi
this is what i got when launched docker

​
Then I got this

​
When I go to http://192.168.99.100:3000, I got blank page. Viewing the
source form the browser it actually call the page correctly but didnt
display/i cant see anything on screem

When I go to http://192.168.99.100:3001 for the UI i got error

​
When run the Android APP, It registers ok, i can send and saw the message
on terminal but not to the mobile.


​
Cant figure out why it didnt display the option on the web for me to send
message to mobile thru GCM


Kind regards;
Hj Azaman Hj Suleiman

**Comment by silvolu** [comment]
@azaman2204 I asked if you could see those lines just to make sure that the UI was serving, but you shouldn't use those IPs. If your IP is 192.168.99.100, then the UI should be served at  http://192.168.99.100:3000.
Which version of Chrome are you using? If you open the Developer Console on the blank page, do you see any errors?

**Comment by azaman2204** [comment]
Yes I use my ip as per docker info when it booted up my defult container.
i.e t http://192.168.99.100:3000. whuch I didnt get any UI. it's just a
blank page.

I'm using this version of Chrome Version 46.0.2490.80 (64-bit)
when I open teh Developer Console I just got blank page, no errors


Kind regards;
Hj Azaman Hj Suleiman

**Comment by wjarek** [comment]
have you tried deploying on GCP or AWS?

**Comment by sanjaylod493** [comment]
https://youtu.be/iJQJ_AiTCjs

**Comment by sanjaylod493** [comment]
https://youtu.be/fJ1_iiiqDAE

**Comment by tbremer19** [comment]
I have exactly the same problem. The browsersync ui shows correctly but when I try to open the playground I only see a blank page.
Test system is Ubuntu 15.10 64bit

**Comment by GorG1993** [comment]
I had the same problem. I ran the commands from the start.sh manually and then it worked. The problem was that because of some unknown error your start.sh shell script did not run entirely and your bower was not installed(that is why you were not seeing anything on google chrome).

**Comment by silvolu** [comment]
#34 should fix this by moving all the requirements to the docker image, let me know if you have any opinions on the change!
